profile/detect_test.go

```diff
@@ -72,6 +72,14 @@ func TestDetectPassesWithProfileScript(t *testing.T) {
 	Expect(os.WriteFile(filepath.Join(ctx.ApplicationPath, ".profile"), []byte(`echo "Hello World!"`), 0600))
 
 	Expect(profile.Detect(ctx)).To(Equal(libcnb.DetectResult{
-		Pass: false, // TODO: this should be true, after implemented
+		Pass: true,
+		Plans: []libcnb.BuildPlan{{
+			Provides: []libcnb.BuildPlanProvide{{
+				Name: "profile",
+			}},
+			Requires: []libcnb.BuildPlanRequire{{
+				Name: "profile",
+			}},
+		}},
```

Comment:
I don't think the buildpack actually needs any provides/requires?

Reply:
so the plans should be empty?

Reply:
This question has come up before. I know that in Paketo we always put something, even if provides/requires match-up like here. I don't totally know why that is though. If it's historic reasons or if there's some function to it. It logically seems redundant though, since you are providing your own requirement. The only thing that I can think of off the top of my head that this would do is to ensure there's an entry in the buildplan, but this buildpack isn't doing that so idk. 

@gcemaj did you by chance try it with an empty buildplan? Did it work the same?

Reply:
i have not yet, can i do it locally?